.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- Please provide a brief summary of your changes: -->
+
+<!-- Please ensure you’ve done all of these things (if applicable). -->
+<!-- You can replace the `[ ]` with `[x]` to mark each task as done. -->
+
+- [ ] I’ve added tests to confirm my change works.
+- [ ] I’ve updated the snapshot files (if changed) and there are no regressions. (`yarn test -u`)
+- [ ] (if changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
+- [ ] I’ve run `yarn lint` to ensure my code is formatted properly.
```

Comment:
I think we could remove this kind of items since they can be caught by CI, and just left those that need people to check it manually. But it can be also a reminder so I'm not sure, what do you think?